core/store/models/job_spec.go

```diff
@@ -237,6 +237,9 @@ func (i *InitiatorParams) SetDefaultValues(typ string) {
 	if typ == InitiatorFluxMonitor {
 		err := mergo.Merge(i, &FluxMonitorDefaultInitiatorParams)
 		logger.PanicIf(errors.Wrap(err, "type level dependent error covered by tests"))
+		if i.PollingInterval.IsInstant() { // mergo can't handle structs well
```

Comment:
What is mergo?

Reply:
And why is it relevant? This comment could be clearer

Reply:
[Mergo](https://github.com/imdario/mergo) magically merges recursive data structures, replacing empty fields in the destination with the values in the source. The `mergo.Merge` call ought to replace `i.PollingInterval` with the default value, but it doesn't.

`mergo` doesn't accomplish much, and I am inclined to rip it out.